src/BobApi/BobApiClient.cs

```
@@ -63,27 +65,46 @@ public async Task<List<Disk>> GetDisksToMonitor()
 
         public async Task<List<Disk>> GetInactiveDisks()
         {
-            var disks = await GetDisksToMonitor();
+            var disks = await GetDisks();
             if (disks != null)
                 disks.RemoveAll(d => d.IsActive);
             return disks;
         }
 
-        public async Task<bool> StartDisk(string name)
+        public async Task<bool> StartDisk(string name, CancellationToken cancellationToken = default)
         {
-            using (var response = await client.PostAsync($"disks/{name}/start", new StringContent("")))
-                return response.IsSuccessStatusCode;
+            try
+            {
+                using (var response = await client.PostAsync($"disks/{name}/start", new StringContent(""), cancellationToken))
+                    return response.IsSuccessStatusCode;
+            }
+            catch
```

It is bad to catch all exceptions. Currently, even `OperationCanceledException` will be blocked from propagation. 
Possible exceptions from `PostAsync` can be found here: https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpclient.postasync?view=net-5.0
Please, catch only them